platform/mellanox/docker-saiserver-mlnx/Dockerfile.j2

```diff
@@ -40,7 +40,7 @@ python-debs/{{ deb }}{{' '}}
 {%- endfor -%}
 debs/
 
-RUN apt-get install -y --no-install-recommends libxml2 iptables libbsd0 protobuf-c-compiler protobuf-compiler python3-protobuf libprotobuf-c1 python3-future python3-ipaddr libnet1 pkg-config asciidoc xmlto
```

Comment:
Why is pytho3-ipaddr removed?

Reply:
This is deprecated in bookworm https://packages.debian.org/bullseye/python3-ipaddr

Besides, the ipaddress lib is already part of python3.11

```
vkarri@sonic-server:/sonic-buildimage$ docker ps -a
CONTAINER ID   IMAGE                            COMMAND                  CREATED          STATUS                      PORTS     NAMES
dcb26195790d   docker-saiserverv2-mlnx:latest   "/usr/local/bin/supe…"   44 seconds ago   Up 43 seconds                         upbeat_rosalind

vkarri@r-build-sonic06:/builds2/vkarri/sonic/sonic-buildimage$ docker exec -it upbeat_rosalind bash

root@dcb26195790d:/# python3
Python 3.11.2 (main, Mar 13 2023, 12:18:29) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import ipaddress
>>> 
```